vyper/lll/README.md

```diff
@@ -299,6 +299,9 @@ Compare or equal
 
 `(ne x y)` is equivalent to `(iszero (eq x y))`
 
+### SELECT
+`(select cond x y)` is equivalent to `(if cond x y)` but it may evaluate both branches. It is analogous to LLVM `select` and is intended to compile to branchless code.
```

Comment:
```suggestion
`(select cond x y)` is similar to `(if cond x y)` but it may evaluate both branches. It is analogous to LLVM `select` and is intended to compile to branchless code.
```